modules/ztp-validating-the-generation-of-installation-crs.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-validating-the-generation-of-installation-crs_{context}"]
+= Validating the generation of installation CRs
+
+SiteConfig applies Installation custom resources (CRs) to the hub cluster in a namespace with the name matching the site name. To check the status, use the following command:
+
+[source,terminal]
+----
+$ oc get AgentClusterInstall -n <clusterName>
+----
+
+If no object is returned, use the following procedure to troubleshoot the ArgoCD pipeline flow from SiteConfig to the installation CRs.
+
+.Procedure
+
+. Check the synchronization of the SiteConfig to the hub cluster using either of the following commands:
```

Comment:
I would suggest checking with SME and QE and provide only the preferred command.

Reply:
@juphoff - WDYT?